controllers/placementrule/hub_info_secret.go

```diff
@@ -38,9 +39,15 @@ func newHubInfoSecret(client client.Client, obsNamespace string,
 	if !strings.HasPrefix(url, "http") {
 		url = protocol + url
 	}
+	hubRouterCA, err := config.GetRouterCA(client)
+	if err != nil {
+		log.Error(err, "Failed to CA of openshift Route")
+		return nil, err
+	}
 	hubInfo := &HubInfo{
 		ClusterName: clusterName,
 		Endpoint:    url + urlSubPath,
+		HubRouterCA: hubRouterCA,
```

Comment:
do we also need to add the url for alertmanagaer route here?

Reply:
yes, we can.

Reply:
added.